draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1180,94 +1131,51 @@ priority = (2^24)*(type preference) +
-candidates obtained from a VPN interface.  When media is transited
-through a media intermediary, it can have a positive or negative
-effect on the latency between transmission and reception.  It may or
-may not increase the packet losses, because of the additional router
-hops that may be taken.  It may increase the cost of providing
-service, since media will be routed in and right back out of a media
-intermediary run by a provider.  If these concerns are important, the
-type preference for relayed candidates must be carefully chosen. The
-RECOMMENDED values are 126 for host candidates, 100 for server
-reflexive candidates, 110 for peer reflexive candidates, and 0 for
-relayed candidates.</t>
-
-<t>Furthermore, if an agent is multihomed and has multiple IP
-addresses, the recommendation in <xref target=
-"I-D.ietf-ice-dualstack-fairness"/> should be followed.  If multiple
+<t>The RECOMMENDED values for type preferences are 126 for host
+candidates, 110 for peer reflexivec andidates, 100 for server
+reflexive candidates, and 0 for relayed candidates.</t>
+
+<t>If an agent is multihomed and has multiple IP
+addresses, the recommendations in <xref target=
+"I-D.ietf-ice-dualstack-fairness"/> SHOULD be followed.  If multiple
 TURN servers are used, local priorities for the candidates obtained
 from the TURN servers are chosen in a similar fashion as for
 multihomed local candidates: the local preference value is used to
 indicate a preference among different servers but the preference MUST
 be unique for each one.</t>
 
-<t>Another criterion for selection of preferences is IP address
-family.  ICE works with both IPv4 and IPv6.  It provides a transition
-mechanism that allows dual-stack hosts to prefer connectivity over
-IPv6, but to fall back to IPv4 in case the v6 networks are
-disconnected. Implementation should follow the guidelines from <xref
-target= "I-D.ietf-ice-dualstack-fairness"/> to avoid excessively delays
-in the connectivity check phase if broken paths exist.</t>
-
-<t>Another criterion for selecting preferences is topological
-awareness. This is most useful for candidates that make use of
-intermediaries. In those cases, if an agent has preconfigured or
-dynamically discovered knowledge of the topological proximity of the
-intermediaries to itself, it can use that to assign higher local
-preferences to candidates obtained from closer intermediaries.</t>
-
-<t>Another criterion for selecting preferences might be security or
-privacy.  If a user is a telecommuter, and therefore connected to a
-corporate network and a local home network, the user may prefer their
-voice traffic to be routed over the VPN or similar tunnel in order to
-keep it on the corporate network when communicating within the
-enterprise, but use the local network when communicating with users
-outside of the enterprise.  In such a case, a VPN address would have a
-higher local preference than any other address.</t>
```

Comment:
for IETF 99 session
The old version had much more details, but perhaps they are not needed.